RELEASE-NOTES.md

```diff
@@ -1,14 +1,18 @@
 # Wikibase DataModel Serialization release notes
 
+## 2.6.0 (2017-09-18)
+
+* Added compatibility with DataValues Number 0.9
+
 ## 2.5.0 (2017-08-30)
 
 * Removed MediaWiki integration files
 * Added compatibility with DataValues Geo 2.x
 * Deprecated `SerializerFactory` options
   `OPTION_SERIALIZE_MAIN_SNAKS_WITHOUT_HASH`,
   `OPTION_SERIALIZE_QUALIFIER_SNAKS_WITHOUT_HASH` and
-  `OPTION_SERIALIZE_REFERENCE_SNAKS_WITHOUT_HASH`,
-  and added `OPTION_SERIALIZE_SNAKS_WITHOUT_HASH` instead
+  `OPTION_SERIALIZE_REFERENCE_SNAKS_WITHOUT_HASH`.
+* Added `SerializerFactory::OPTION_SERIALIZE_SNAKS_WITHOUT_HASH`.
```

Comment:
Just leaving a note that this change does not really seem related/necessary to me but whatever.